Libraries/LibTLS/TLSv12.h

```diff
@@ -468,6 +471,8 @@ class TLSv12 : public Core::Socket {
     OwnPtr<Crypto::Cipher::AESCipher::CBCMode> m_aes_remote;
 
     bool m_has_scheduled_write_flush = false;
+
+    i32 m_max_wait_time_for_handshake = 10; // seconds
```

Comment:
Please use `{}` for member variable initialization.
Also, let's encode the "// seconds" in the variable name, something like `m_max_time_to_wait_for_handshake_in_seconds`.

Reply:
Of course.

Reply:
It's just that the line above used `=`, so I stuck to the local theme.
Should I change that too?

Reply:
Please :)